samples/README.md

```diff
@@ -49,7 +49,7 @@ Commands:
 Options:
   --version        Show version number                                                                         [boolean]
   --help           Show help                                                                                   [boolean]
-  --projectId, -p                                                                [string] [default: "precise-truck-742"]
```

Comment:
This is weird. Why does this have a default at all? And why was it generated with what happened to be my environment variable? (Do we have a bug to track this?)